tools/autograd/derivatives.yaml

```diff
@@ -1202,18 +1223,23 @@
 
 - name: sqrt(Tensor self) -> Tensor
   self: grad / (2 * result.conj())
+  result: auto_element_wise
 
 - name: squeeze(Tensor(a) self) -> Tensor(a)
   self: unsqueeze_to(grad, self.sizes())
+  result: auto_linear
 
 - name: squeeze.dim(Tensor(a) self, int dim) -> Tensor(a)
   self: unsqueeze_to(grad, dim, self.sizes())
+  result: auto_linear
 
 - name: squeeze_(Tensor(a!) self) -> Tensor(a!)
   self: unsqueeze_to(grad, self.sizes())
+  result: self_t.squeeze()
```

Comment:
Should this be in-place squeeze to match? (I'm not sure)

Reply:
Or in general for auto-linear inplace ops, can we avoid allocating the new self_t and modify the old self_t in place?

Reply:
Yes this should be linear so that it is inplace.